docker-compose.yaml

```diff
@@ -2,12 +2,13 @@ services:
   postgres:
     image: postgres
     volumes:
-      - ./karyakram:/var/lib/postgresql/data
+      # - ./karyakram:/var/lib/postgresql/data
+      - E:\Install Software\PostgreSQL\15\data:/var/lib/postgresql/data
```

Comment:
added to gitignore for now

Reply:
docker-compose.yaml file is still there. 
Once the file is committed in git it needs to be deleted then only .gitignore will ignore that file. This file is still here which is what we want in this case. We will need this file so I'm okay with commenting it out instead of keeping it in the .gitignore file so as not to block the PR approval.